ui/app/AppLayouts/Profile/Sections/BackupSeedModal.qml

```diff
@@ -0,0 +1,117 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import "../../../../imports"
+import "../../../../shared"
+
+ModalPopup {
+    id: popup
+
+    title: qsTr("Write down your seed phrase")
+
+    Item {
+
+      id: seed
+      anchors.left: parent.left
+      anchors.right: parent.right
+      width: parent.width
+      height: children[0].height
+
+      Component {
+        id: wordCmp
+
+        Item {
+          height: 40
+          anchors.left: parent.left
+          anchors.leftMargin: Theme.bigPadding
+
+          TextEdit {
+            text: word
+            font.pixelSize: 15
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: Theme.smallPadding
+            selectByMouse: true
+            readOnly: true
+          }
+        }
+      }
```

Comment:
This Item doesn't include the "count" next to the word (like `1. word1 2. word2 ...`). Can't rely on `currentIndex` because we're having two `ListViews` here. Wanted to do some funny things with a custom counter but I think that's not the right way either..

Reply:
If you handle it like a `string` like I suggested above, it's gonna work